lib/checks/canFindDenominatorInNumerator.js

```diff
@@ -0,0 +1,35 @@
+const Node = require('../node');
+
+// Returns true if by adding a term you can simplify part of the function into an integer
+// e.g. (2x+1)/(2x+3) -> True
```

Comment:
it would be easiest to understand this function (from an outsider perspective) if you explain why it's true as well

```
// e.g. (2x+1)/(2x+3) -> True 
// because of the simplification  (2x+1)/(2x+3) -> (2x + 3)/(2x + 3) - 2/(2x + 3) -> 1 - 2/(2x + 3) 
```